taskcluster/scripts/pipeline/train-taskcluster.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -x
+set -euo pipefail
+
+[[ -v MOZ_FETCHES_DIR ]] || { echo "MOZ_FETCHES_DIR is not set"; exit 1; }
+[[ -v VCS_PATH ]] || { echo "VCS_PATH is not set"; exit 1; }
+
+if [ "$#" -lt 10 ]; then
+    echo "Usage: $0 <model_type> <training_type> <src_locale> <trg_locale> <train_set_prefix> <valid_set_prefix> <model_dir> <best_model_metric> <pretrained_model_mode> <pretrained_model_type> [extra_params...]"
+    exit 1
+fi
+
+model_type=$1
+training_type=$2
+src=$3
+trg=$4
+train_set_prefix=$5
+valid_set_prefix=$6
+model_dir=$7
+best_model_metric=$8
+pretrained_model_mode=$9
+pretrained_model_type=${10}
+extra_params=( "${@:11}" )
+
+if [ "$pretrained_model_mode" == "None" ]; then
+    vocab="$MOZ_FETCHES_DIR/vocab.spm"
+else
+    vocab="$TASK_WORKDIR/artifacts/vocab.spm"
```

Comment:
question: I want to make sure I'm understanding this correctly. If we're using a pretrained model, we pull the vocab from a mount? And if not, we pull it from a fetch?

That all seems fine, but if this is true, I would prefer to see the fetch and dependency on `train-vocab` not happening at all if we're using a pretrained model. It's not a large file, but it makes it much less clear where the vocab comes from if we still maintain this dependency while not using anything from it. Please make sure there's a sanity check that ensure that either a pretrained model being used OR a dependency on `train-vocab` in this case.

Reply:
Mm I can probably do something regarding that in training_continuation.py

Reply:
I refactored the code so we don't have the fetch and dependency on `train-vocab` if the `vocab.spm` file is mounted.